kong/plugins/aws-lambda/handler.lua

```diff
@@ -1,39 +1,24 @@
 -- Copyright (C) Kong Inc.
 local BasePlugin = require "kong.plugins.base_plugin"
 local aws_v4 = require "kong.plugins.aws-lambda.v4"
-local singletons = require "kong.singletons"
-local responses = require "kong.tools.responses"
-local constants = require "kong.constants"
-local utils = require "kong.tools.utils"
-local meta = require "kong.meta"
 local http = require "resty.http"
 local cjson = require "cjson.safe"
-local public_utils = require "kong.tools.public"
 
 
 local tostring             = tostring
 local tonumber             = tonumber
-local pairs                = pairs
 local type                 = type
 local fmt                  = string.format
-local ngx                  = ngx
-local ngx_req_read_body    = ngx.req.read_body
-local ngx_req_get_uri_args = ngx.req.get_uri_args
-local ngx_req_get_headers  = ngx.req.get_headers
 local ngx_encode_base64    = ngx.encode_base64
 
 
-local new_tab
-do
-  local ok
-  ok, new_tab = pcall(require, "table.new")
-  if not ok then
-    new_tab = function(narr, nrec) return {} end
-  end
-end
-
-
-local server_header = meta._SERVER_TOKENS
```

Comment:
So much red! This is great :grin: The updated plugins with PDK usage look lighter and easier to read!